annotations/helm-annotations/src/main/java/io/dekorate/helm/listener/HelmWriterSessionListener.java

```diff
@@ -85,6 +85,7 @@ public void onClosed() {
     Session session = getSession();
     Project project = getProject();
     Path outputDir = project.getBuildInfo().getClassOutputDir().resolve(project.getDekorateOutputDir());
+    System.out.println("Project name: " + project.getBuildInfo().getName());
```

Comment:
I think we need to remove this or switch to the Logger.

Reply:
Ops, removed!